validator/validator_test.go

```diff
@@ -32,15 +32,15 @@ func TestValidator_ValidateToken(t *testing.T) {
 		algorithm       string
 		customClaims    CustomClaims
 		expectedError   error
-		expectedContext *UserContext
+		expectedContext *ValidatedClaims
```

Comment:
We should update the name of the struct field `expectedContext`.

Reply:
Fixed in https://github.com/auth0/go-jwt-middleware/pull/121/commits/e0cfb1f971d7b143f6a20d415a5554e91979068a